SYCL/Basic/accessor/Inputs/host_task_accessor.cpp

```diff
@@ -54,9 +54,6 @@ int main() {
 
 #if defined(accessor_new_api_test) || defined(buffer_new_api_test)
```

Comment:
Is this if-statement needed after the removal of codeplay_host_task?

Reply:
All of the accessors are defined under #ifdef so they are not available without this check